pkg/apis/componentconfig/v1alpha1/register.go

```diff
@@ -45,6 +43,8 @@ func init() {
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&KubeSchedulerConfiguration{},
+		&KubeControllerManagerConfiguration{},
```

Comment:
don't add these here, we're moving them out in a minute :)

Reply:
may be a mistakes for code quickly